src/shogun/classifier/svm/LibSVMOneClass.h

```diff
@@ -32,11 +32,8 @@ class CLibSVMOneClass : public CSVM
 		CLibSVMOneClass(float64_t C, CKernel* k);
 		virtual ~CLibSVMOneClass();
 
-		/** get classifier type
-		 *
-		 * @return classifier type LIBSVMONECLASS
-		 */
-		virtual EMachineType get_classifier_type() { return CT_LIBSVMONECLASS; }
+		/** @return whether this svm is one-class or multi-class */
```

Comment:
minor maybe change to "one-class or not"
(since we can have binary and multi class)